Shoko.Server/API/v2/Modules/PlexWebhook.cs

```diff
@@ -241,74 +245,183 @@ private T CallPlexHelper<T>(Func<PlexHelper, T> act)
             return act(PlexHelper.GetForUser(user));
         }
 
-        #region plexapi
-        public class PlexEventSuper
-        {
-            public string user { get; set; }
-            public string Owner { get; set; }
-        }
-#pragma warning disable 0649
+        #region Plex Event Models
+        #pragma warning disable 0649
+
         public class PlexEvent
         {
             [Required]
+            [DataMember(Name = "event")]
             public string Event;
+
+            [DataMember(Name = "user")]
             public bool User;
+
+            [DataMember(Name = "owner")]
             public bool Owner;
 
             [Required]
+            [DataMember(Name = "Account")]
             public PlexAccount Account;
+
             [Required]
+            [DataMember(Name = "Server")]
             public PlexBasicInfo Server;
+
+            [DataMember(Name = "Player")]
             public PlexPlayerInfo Player;
+
+            [DataMember(Name = "Metadata")]
             [Required]
             public PlexMetadata Metadata;
 
+            [DataContract]
             public class PlexAccount
             {
+                [DataMember(Name = "id")]
                 public int Id;
-                public string Thumb;
+
+                [DataMember(Name = "thumb")]
+                public string Thumbnail;
```

Comment:
Somewhat of a pointless change - though this I won't say needs to be changed back.

Reply:
:+1: